frontend/src/utils/chart/TreeMap.js

```diff
@@ -1,7 +1,7 @@
 import { Easing, backgroundColor } from "./chart-style.js";
 
 const TreeMap = (data, extra, selected) => {
-  let color = [
+  const color = [
```

Comment:
same here, style constants should be centralised in one place, not spreaded along the whole app

Reply:
moving it to `chart-style.js`